openc3/spec/interfaces/http_client_interface_hal_spec.rb

```diff
@@ -0,0 +1,151 @@
+
```

Comment:
What is the `hal` in the file name for? We already have a `http_client_interface_spec.rb`. These additional methods should just go in there.

Reply:
Yes, done.